go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21
 
 require (
 	github.com/confluentinc/confluent-kafka-go/v2 v2.3.0
+	github.com/deckarep/golang-set/v2 v2.1.0
```

Comment:
Does this library do anything sophisticated? Or could we use `map[T]struct{}` with `maps.Contains`/`maps.Keys`?

Reply:
I'm using it for thread safe `Set` not for `map`

Reply:
The idiomatic representation for a set in Go is a map with either `struct{}` or `bool` values.
You can see the implementation of the module imported here is doing the same thing: https://github.com/deckarep/golang-set/blob/main/threadunsafe.go#L34

There are interesting set operations in this library, but we don't seem to be using them, so the import may not be worth it. Although more importantly, it doesn't typically make sense to send types like this over grpc, since we want to pass data, not provide shared access to the raw data structures that we are holding on to.